### DISCUSSION TITLE
GERS in transportation theme roadmap

### DISCUSSION THREAD

**Discussion by stemillington-flock** [OvertureMaps/data#82] [Discussion]
Is there any current planned timeframe for having stable(-ish!) GERS format for the transportation theme, and when could we expect to see some data published?

**Comment by vcschapp** [comment]
Great question! My apologies for the lengthy turnaround time, but we believe that as of the January 17, 2024 data release, GERS IDs for the transportation theme should be stable(ish). [Release Notes for 1/17/2024 release](https://overturemaps.org/overture-january-2024-release-notes/).

To share a bit more detail, from the January release onward we are including "mostly stable" GERS IDs. Consumers should notice this stability from the February release onwards, since the stable IDs only began to be assigned in January. Throughout February we will be continuing development work to eliminate some known cases of **overstability** (ID reused when it should not have been) and **understability** (ID thrown away when it should not have been). We expect that by March, these over/under issues should be resolved so that from March to April and thereafter, data consumers should see Pretty Good Stability (TM).
